### ISSUE TITLE
Docker image for client not building correctly

### ISSUE THREAD

**Issue by zirpins** [di-unipi-socc/microFreshener#8]
Example output:

```
$ docker-compose build client
WARNING: Some networks were defined but are not used by any service: internal
Building client
Step 1/12 : FROM node:11.4.0-alpine as builder
 ---> f21b938fb1e6
Step 2/12 : RUN mkdir /app
 ---> Using cache
 ---> 7d45a6c3a539
Step 3/12 : WORKDIR /app
 ---> Using cache
 ---> bbef20ecc135
Step 4/12 : COPY package.json package-lock.json /app/
 ---> Using cache
 ---> 048380a64b82
Step 5/12 : RUN npm install
 ---> Using cache
 ---> 931cb711001b
Step 6/12 : COPY . /app
 ---> Using cache
 ---> 22a57a63e991
Step 7/12 : ARG configuration=production
 ---> Using cache
 ---> 216d0e0adabd
Step 8/12 : RUN npm run build -- --output-path=./dist/out --configuration $configuration
 ---> Running in 802a59451260

> microfreshener-client@1.0.2 build /app
> ng build "--output-path=./dist/out" "--configuration" "production"

An unhandled exception occurred: Could not find the implementation for builder @angular-devkit/build-angular:browser
See "/tmp/ng-OmNNFL/angular-errors.log" for further details.
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! microfreshener-client@1.0.2 build: `ng build "--output-path=./dist/out" "--configuration" "production"`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the microfreshener-client@1.0.2 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-11-04T15_17_53_944Z-debug.log
ERROR: Service 'client' failed to build: The command '/bin/sh -c npm run build -- --output-path=./dist/out --configuration $configuration' returned a non-zero code: 1
```

Possible solution: fix angular builders dependency in package.json:

"devDependencies": {
    "@angular-devkit/build-angular": "^0.803.17",
    ...
}

**Comment by Mastercuber** [comment]
Like you observed there is a version conflict. `npm uninstall @angular-devkit/build-angular` and then `npm install @angular-devkit/build-angular --save-dev` should do the trick.

**Comment by dido18** [comment]
Thank you @zirpins @Mastercuber  for having opened the issues.

The issue should be resolved in commit 4342843

I have also updated the README for running the application with Docker compose.
